### ISSUE TITLE
Add hidden labels to fieldsets

### ISSUE THREAD

**Issue by ajb** [dobtco/dvl-core#244]
_From @jrubenoff on March 2, 2016 23:34_

Each input in a fieldset should have a label for accessibility. If the label isn't visible in the UI, we should hide it with the `hidden` mixin. Example:

![](https://cloud.githubusercontent.com/assets/1328849/12955904/1496ac1c-cfda-11e5-9b86-48d54168c545.png)

_Copied from original issue: dobtco/screendoor-v2#3310_

**Comment by ajb** [comment]
We will use `aria-label` for this. I can add a pattern to the knowledge base?

**Comment by jrubenoff** [comment]
Don't we already have this with the date and item picker? You can add a `hint` describing that change if you want.

This is primarily a Screendoor task, really... making sure we have fieldsets with `aria-label` everywhere.

**Comment by ajb** [comment]
K. I'll open an issue in Screendoor and close this.
